devdocs/settingUp.md

```diff
@@ -100,9 +100,7 @@ Important: When a version is specified, please install that version instead of t
     Firefox 38.0.5 (latest release as at 7th June 2015) is supported.
    
 2. Before running the test suite, both the server and the test environment 
-   should be using the UTC time zone. The server and the test environment should 
-   also serve the CDN files (files that we off-load to servers such as Google's servers)
-   locally instead. In our case, these are files such as jQuery.min.js.
+   should be using the UTC time zone.
```

Comment:
Ensure that the CDN trick is mentioned somewhere in the code. Otherwise future devs will wonder why we switch locations of the jquery files.